packages/flutter/lib/src/material/bottom_app_bar_theme.dart

```diff
@@ -9,20 +9,24 @@ import 'package:flutter/widgets.dart';
 
 import 'theme.dart';
 
-/// Defines a theme for [BottomAppBar] widgets.
+/// Defines default property values for descendant [BottomAppBar] widgets.
 ///
 /// Descendant widgets obtain the current [BottomAppBarTheme] object using
 /// `BottomAppBarTheme.of(context)`. Instances of [BottomAppBarTheme] can be
 /// customized with [BottomAppBarTheme.copyWith].
 ///
+/// Typically a [BottomAppBarTheme] is specified as part of the overall [Theme]
+/// with [ThemeData.bottomAppBarTheme].
+///
+/// All properties are `null` by default. When null, the [BottomAppBar] constructor
```

Comment:
Since "all properties" is now a little confusing: All [BottomAppBarTheme] properties are ...

Reply:
Done.